native/packages/react-native-bpk-component-panel/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "react-native-bpk-component-panel",
+  "version": "1.2.12",
+  "main": "index.js",
+  "description": "Backpack React Native panel component.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^16.0.0-alpha.1",
+    "react-native": ">= 0.47.0"
+  },
+  "dependencies": {
+    "bpk-tokens": "^26.7.4",
+    "prop-types": "^15.5.8",
+    "react-native-bpk-component-touchable-native-feedback": "^1.1.0",
+    "react-native-bpk-component-touchable-overlay": "^1.0.13",
```

Comment:
The two toucheable are not needed right? it's just a view with children, doesn't have touchable behaviuours?

Reply:
💯